Ideosynergy/worksfair-backend
# #163348981 Setup django and coding environment

#### What does this PR do?
- Setup django and coding environment

#### Description of Task to be completed?
- Install django
- Setup postgresql database
- setup superadmin

#### How should this be manually tested?
- git clone `https://github.com/Ideosynergy/worksfair-backend.git`.
- Install `python` if not installed.
- ensure that PostgreSQL is running on your local machine.

To test
- run `python3 worksfair/manage.py runserver`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#163348981

#### Screenshots (if appropriate)
